sonarts-core/jest.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   },
   mapCoverage: true,
   moduleFileExtensions: ["js", "ts"],
+  testResultsProcessor: "jest-sonar-reporter",
```

Comment:
[configure](https://github.com/3dmind/jest-sonar-reporter#customization) it to generate report in `coverage/`folder

Reply:
or add `test-report.xml` to `.gitignore`, but I'd prefer the first option

Reply:
@stas-vilchik for some reason it didn't work for "coverage" directory, may be dir was recreated for coverage

Reply:
Added file to gitignore